docs/en/Time-To-Live.md

```diff
@@ -0,0 +1,114 @@
+might not be enforced until the next check interval, ie. short TTL intervals can take up to
+two hours to be enforced. The interval length is set by the `alluxio.master.ttl.checker.interval`
+property.
+
+For example, to set the interval to 10 minutes, add the following to alluxio-site.properties:
+
+```
+alluxio.master.ttl.checker.interval=10m
+```
+
+Refer to the [configuration page](Configuration-Settings.html) for more details on setting Alluxio
+configurations.
+
+While the master node enforces TTLs, it is up to the clients to set the appropriate TTLs.
+
+## APIs
+
+There are three ways to set the TTL of a path.
+
+1. Through the Alluxio shell command line.
+1. Through the Alluxio Java File System API.
+1. Passively on each load metadata or create file.
+
+The TTL API is as follows:
+
+```
+SetTTL(path, duration, action)
+`path` 			the Alluxio path in the namespace
+`duration` 		the number of milliseconds before the TTL action takes into effect, this overrides
+				any previous value
+`action`		the action to take when duration has elapsed. `FREE` will cause the file to be
+				evicted from Alluxio storage, regardless of the pin status. `DELETE` will cause the
+				file to be deleted from the Alluxio namespace and under store.
+```
+
+### Command Line Usage
+
+See the detailed [command line documentation](Command-Line-Interface.html#setttl).
+
+### Java File System API
+
+Use the Alluxio FileSystem object to set the file attribute with the appropriate options.
+
+```
+FileSystem alluxioFs = FileSystem.Factory.get();
+
+AlluxioURI path = new AlluxioURI("alluxio://hostname:port/file/path");
+long ttlMs = 86400000L; // 1 day
+TtlAction ttlAction = TtlAction.FREE; // Free the file with TTL is hit
+
+SetAttributeOptions options = SetAttributeOptions.defaults().setTtl(ttlMs).setTtlAction(ttlAction);
+alluxioFs.setAttribute(path);
+```
+
+See the [Javadocs](http://www.alluxio.org/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/index.html) for
+more details.
+
+### Passively on load metadata or create file
+
+Whenever a new file is added to the Alluxio namespace, the user has the option of passively adding
+a TTL to that file. This is useful in cases where files accessed by the user are expected to be
+temporarily used. Instead of calling the API many times, it is automatically set on file discovery.
+
+Passive TTL works with the following configuration options:
+
+* `alluxio.user.file.load.ttl` - the default duration to give any file newly loaded into Alluxio from
+an under store. By default this is no ttl.
+* `alluxio.user.file.load.ttl.action` - the default action for any ttl set on a file newly loaded into
+Alluxio from an under store. By default this is `DELETE`.
+* `alluxio.user.file.create.ttl` - the default duration to give any file newly created in Alluxio. By
+default this is no ttl.
+* `alluxio.user.file.create.ttl.action` - the default action for any ttl set on a file newly created
+in Alluxio. By default this is `DELETE`.
```

Comment:
We should add **NOTE** to warn `DELETE` is a dangerous action in production environment since it also deletes data in UFS. User should clearly know what he is doing when `DELETE` set.

Reply:
Good idea, updated.